### PR TITLE
sample notebook `fighting_california_forest_fires_using_spatial_analysis.ipynb` updated to incorporate the new map widget

### PR DESCRIPTION
**Pull request by tarunkukreja003**
sample notebook `fighting_california_forest_fires_using_spatial_analysis.ipynb` updated to incorporate the new map widget

please use the playground portal account and api_data_owner account for creating the GIS object since the data is stored there

### CODE REVIEW

File: samples/04_gis_analysts_data_scientists/fighting_california_forest_fires_using_spatial_analysis.ipynb [hunk 1]
```diff
@@ -30,7 +30,7 @@
   },
```
Comment (cariashuang0417) [inline samples/04_gis_analysts_data_scientists/fighting_california_forest_fires_using_spatial_analysis.ipynb:30]: <pre class="ql-syntax" spellcheck="false">Line #3.        if webmap.content.layers[0].properties['name'].startswith('At_Risk_Infrastructure'):
</pre><p><code>if lyr.properties['name'].startswith('At_Risk_Infrastructure'):</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2141/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Comment (cariashuang0417) [inline samples/04_gis_analysts_data_scientists/fighting_california_forest_fires_using_spatial_analysis.ipynb:30]: <pre class="ql-syntax" spellcheck="false">Line #4.        if webmap.content.layers[0].properties['name'].startswith('At_Risk_Infrastructure'):
</pre><p>Same as above - <code>﻿if lyr.properties['name'].startswith('At_Risk_Infrastructure'):</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2141/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

**Review by cariashuang0417 — CHANGES_REQUESTED** [review]
Left comments for a little tweak in scripts. Overall looks good to me!

File: samples/04_gis_analysts_data_scientists/fighting_california_forest_fires_using_spatial_analysis.ipynb (hunk 1 above)
Comment (nanaeaubry) [inline samples/04_gis_analysts_data_scientists/fighting_california_forest_fires_using_spatial_analysis.ipynb:30]: <p>Map doesn't take zoom as a parameter. You can move this to a cell below and do <code>m.zoom = 9</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2141/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (nanaeaubry) [inline samples/04_gis_analysts_data_scientists/fighting_california_forest_fires_using_spatial_analysis.ipynb:30]: <p>Same here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2141/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (nanaeaubry) [inline samples/04_gis_analysts_data_scientists/fighting_california_forest_fires_using_spatial_analysis.ipynb:30]: <p>Ditto</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2141/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Comment (nanaeaubry) [inline samples/04_gis_analysts_data_scientists/fighting_california_forest_fires_using_spatial_analysis.ipynb:30]: <p>ditto</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2141/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

**Review by nanaeaubry — CHANGES_REQUESTED** [review]
Small changes with zoom in the initializer

**Review by jyaistMap — CHANGES_REQUESTED** [review]
Please address @nanaeaubry's comments about removing the zoom integer from the _Map_ initialization and adding a cell under the map image that zooms:
```python
m.zoom = 9
```

**Review by jyaistMap — CHANGES_REQUESTED** [review]
Please change the cells @cariashuang0417 references as well.

**Comment by tarunkukreja003** [comment]
thanks for comments/suggestions, they are addressed

**Review by nanaeaubry — APPROVED** [review]

**Review by jyaistMap — APPROVED** [review]
* build website locally and looks good

**Pull request merged**